.changes/next-release/feature-paramfile-57445.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "feature",
```

Comment:
Talked offline, but for anything we are changing defaults or removing functionality, we should label it as `breaking-change` in order to make it easier to identify breaking changes when parsing through the changelog.